modules/core/src/adapter/types/parameters.ts

```diff
@@ -130,8 +132,10 @@ export type BlendFactor =
   | 'dst-alpha'
   | 'one-minus-dst-alpha'
   | 'src-alpha-saturated'
-  | 'blend-color'
-  | 'one-minus-blend-color';
+  | 'constant-color'
+  | 'one-minus-constant-color'
+  | 'constant-alpha'
+  | 'one-minus-constant-alpha';
```

Comment:
⚠️ NOTICE

I don't see `blend-color` in the WebGPU spec, but probably this is what was intended for luma's constants? The exact [GPUBlendFactor values of the WebGPU spec](https://www.w3.org/TR/webgpu/#enumdef-gpublendfactor) are a bit different, so if we meant for this to be a 1:1 mapping to WebGPU, more changes would be needed.

Reply:
I can't check now but generally the goal is to follow webgpu and if support for some webgl-only constant is needed, we suffix it with -webgl.

So... This could just be a simple mistake on my part, but also I started writing some of this code before  the spec was frozen and there have already been a few places where the luma code had to be updated.

Reply:
Ok! It's probably easier to have two Luma parameters map to the same WebGPU constant, than to have the same Luma parameter map to two different WebGL constants depending on context. So I think I slightly prefer this as you have it, with a slight divergence from WebGPU for an easier mapping to WebGL.